View3D/Services/MeshOptimizerService.cs

```diff
@@ -6,25 +6,45 @@
 using System.Text;
 using View3D.Rendering;
 using View3D.Rendering.Geometry;
+using Simplygon;
+using System.Windows.Interop;
+using SharpDX.Direct3D9;
+using System.Reflection;
+using System.Windows.Controls;
 
 namespace View3D.Services
 {
     public class MeshOptimizerService
```

Comment:
This class now breaks the "Single responsibility principle". It is responsible for picking an algorithm and executing two different ones. This means it should be broken up into 3 classes.
MeshOptimizerService => Responsible for picking the alogrith
SimplygonMeshOptimizer => Execute the simplygon flow
SimpleMeshOptimizer => Execute the old flow.